tests/c/blockmap.c

```diff
@@ -1,3 +1,5 @@
+// # The function __yktrace_hwt_mapper_blockmap_len is compile-time guarded for hwt only.
+// ignore-if: test "$YKB_TRACER" != "hwt" && test "$YKB_TRACER" != ""
```

Comment:
Can `YKB_TRACER` ever be the empty string? I think `test "$YKB_TRACER" != "hwt"` is sufficient.

Reply:
Updated 👉 b3119f15916eaf41311e46631b82cc177504a000